test/functionalTest/cases/3842_per_sub_httpTimeout/per_sub_httpTimeout_notification.test

```diff
@@ -0,0 +1,251 @@
+    ]
+  },
+  "notification": {
+    "http": {
+      "url": "http://localhost:'$LISTENER_PORT'/noresponse",
+      "httpTimeout":12000
+    }
+  }
+}'
+orionCurl --url /v2/subscriptions --payload "$payload"
+echo
+echo
+idSubscription1=$(echo "$_responseHeaders" | grep Location | awk -F/ '{ print $4 }' | tr -d "\r\n")
+
+
+
+echo "02. Create a sub with httpTimeout=8000"
+echo "======================================"
+payload='
+{
+  "description": "HTTP sub",
+  "subject": {
+    "entities": [
+      {
+        "id": "E",
+        "type": "T"
+      }
+    ]
+  },
+  "notification": {
+    "http": {
+      "url": "http://localhost:'$LISTENER_PORT'/noresponse",
+      "httpTimeout":8000
+    }
+  }
+}'
+orionCurl --url /v2/subscriptions --payload "$payload"
+echo
+echo
+idSubscription2=$(echo "$_responseHeaders" | grep Location | awk -F/ '{ print $4 }' | tr -d "\r\n")
+
+
+
+echo '03. Create an entity (notification triggered)'
+echo '============================================='
+payload='{
+  "id": "E",
+  "type": "T",
+  "A": {
+    "value": 1,
+    "type": "Number"
+  }
+}'
+orionCurl --url /v2/entities --payload "$payload"
+echo
+echo
+
+# FIXME This test is using /noresponse path on accumulator for delaying the notification's response. This may enlarge
+# the testing time. Implementing a new accumulator route with a shorter delay would improve it
+echo '04. Wait 12 seconds'
+echo '==================='
+sleep 12
+echo
+echo
+
+echo "05. Check if 1st sub notification succeed"
+echo "========================================="
+orionCurl --url "/v2/subscriptions/${idSubscription1}"
+echo
+echo
+
+
+echo "06. Check if 2nd sub notification failed"
+echo "========================================"
+orionCurl --url "/v2/subscriptions/${idSubscription2}"
+echo
+echo
+
+
+
+--REGEXPECT--
+01. Create a sub with httpTimeout=12000
+=======================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+02. Create a sub with httpTimeout=8000
+======================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+03. Create an entity (notification triggered)
+=============================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/entities/E?type=T
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+04. Wait 12 seconds
+===================
+
+
+05. Check if 1st sub notification succeed
+=========================================
+HTTP/1.1 200 OK
+Content-Length: 421
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "HTTP sub", 
+    "id": "REGEX([0-9a-f]{24})", 
+    "notification": {
+        "attrs": [], 
+        "attrsFormat": "normalized", 
+        "http": {
+            "httpTimeout": 12000, 
```

Comment:
`httpTiemout` inside an element that already is named `http` it somehow redundant. I'd suggest to use just `timeout` instead so we can read things like this:

```
 "http": {
            "timeout": 12000, 
             "url": "http://localhost:9997/noresponse"
```

Reply:
Done in https://github.com/telefonicaid/fiware-orion/pull/3948/commits/f21c65f82654db155e0e2835387d0d531572d617